### PR TITLE
Feature brute force

### PR DESCRIPTION
**Pull request by lyakhovka**
In this branch I have added brute force implementation: an approach that finds all the possible solutions to find a satisfactory solution to a given problem.

- The constructor with no key value is introduced to 'Secret' class.
- Method 'bruteForce()' is implemented. It runs decoding for all possible keys and stores the result in the file.

### CODE REVIEW

File: Secret.java [hunk 1]
```diff
@@ -73,10 +87,23 @@ public void testEncoding() {
         System.out.println("Secret revealed: " + res);
     }
 
+    //This method runs decoding of the source file content for every possible key. All results are stored in the 'decodedFilePath'.
+    // Evaluation of which result makes sense is up to user.
+    public void bruteForce(){
+        Cezar cypher = new Cezar();
+        for(int i=0; i< cypher.getAlphabetLength(); i++){
```
Comment (Volodymyr95) [inline Secret.java:94]: formatting

File: Secret.java [hunk 2]
```diff
@@ -73,10 +87,23 @@ public void testEncoding() {
...
+    }
+
     private void storeResult(String res, Path filePath) {
 
         try {
-            Files.writeString(filePath, res, StandardCharsets.UTF_8);
+            Files.writeString(filePath, res, CREATE, APPEND);
+
         } catch (IOException e) {
             throw new RuntimeException(e);
```
Comment (Volodymyr95) [inline Secret.java:108]: don't re-thorw wxception, add informative message for user

File: runner.java [hunk 3]
```diff
@@ -3,9 +3,8 @@
 public class runner {
     public static void main(String[] args) {
 
-        Secret secret = new Secret("/home/olena/work/cezar/secret", 15);
-        secret.encode();
-        secret.testEncoding();
+        Secret secret = new Secret("/home/olena/work/cezar/secret");
```
Comment (Volodymyr95) [inline runner.java:6]: use url from input parameters

File: Secret.java (hunk 1 above)
Comment (lyakhovka) [reply Secret.java:94]: fixed

File: runner.java (hunk 3 above)
Comment (lyakhovka) [reply runner.java:6]: fixed

File: Secret.java (hunk 2 above)
Comment (lyakhovka) [reply Secret.java:108]: fixed

**Pull request merged**